src/components/FileUpload/index.jsx

```diff
@@ -93,15 +94,18 @@ const FileUpload = ({
         columns={columns}
       />
       {!isReadOnly && fileUploadLimit > uploadedFiles.length && (
-        <Dropzone
-          multiple
-          onProcessUpload={onProcessUpload}
-          progressVariant="bar"
-          accept={{
-            '*': (allowedExtensions || []).map((ext) => `.${ext}`),
-          }}
-          maxSize={maxFileSize}
-        />
+        <Form.Group isInValid>
+          <Dropzone
+            multiple
```

Comment:
I know that you didn't change this, but to my knowledge the paragon version of dropzone does not support multiple files. I would double check this functionality and remove the attribute if multiple files are not supported.

Reply:
Correct. I attempted to upload two files at a time but came across this error `Only one upload permitted.`